### PR TITLE
site: Add v1.11 blog post

### PR DESCRIPTION
**Pull request by stevesloka**
Signed-off-by: Steve Sloka <slokas@vmware.com>

### CODE REVIEW

File: site/_posts/2020-12-18-contour_v1110.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+```
+
+The `Contour` is now ready to start serving `Ingress` and `HTTPProxy` resources.
+
+Check out the [Getting Started](https://projectcontour.io/getting-started/) guide to learn how to start managing Contour using the operator. Learn more about the planned features of Contour Operator by reviewing the GitHub [Issues](https://github.com/projectcontour/contour-operator/issues).
+
+## XDS Resource `v2` Removed
+
+As mentioned in Contour [1.10](https://projectcontour.io/contour_v1100/#envoy-xds-v3-support) the v2 XDS resource version has been removed from Contour ahead of its removal from Envoy.
+Please see the [XDS Migration Guide](https://projectcontour.io/guides/xds-migration/) for upgrading your instances of Envoy/Contour.
```
Comment (sunjayBhatia) [inline site/_posts/2020-12-18-contour_v1110.md:71]: Mention the External Auth service v2 -> v3 change too, as in the release notes?

File: site/_posts/2020-12-18-contour_v1110.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+[Contour Operator](https://github.com/projectcontour/contour-operator/blob/main/README.md) provides a method for packaging,
+deploying, and managing Contour. The operator extends the functionality of the Kubernetes API to create, configure, and
+manage instances of Contour on behalf of users.
+It builds upon the basic Kubernetes resource and controller concepts, but includes domain-specific knowledge to automate the entire lifecycle of Contour.
+
+In Kubernetes, controllers of the control-plane implement control loops that repeatedly compare the desired state of the cluster to its actual state.
+If the cluster's actual state doesn’t match the desired state, then the controller takes action to fix the problem.
+Contour Operator is a custom Kubernetes controller that uses the `Contour` custom resource (CR) to manage Contour and its dependent components, i.e. the Envoy DaemonSet.
+The `contours.operator.projectcontour.io` Custom Resource Definition (CRD) defines the `Contour` CR. A `Contour` is handled by the Kubernetes API just like built-in objects, including interaction via kubectl and inclusion in role-based access control (RBAC) policies.
```
Comment (jonasrosland) [inline site/_posts/2020-12-18-contour_v1110.md:26]: Should that `contours.operator.projectcontour.io` be `contour.operator.projectcontour.io`? I genuinely don't know, so please correct me if I'm wrong.

**Review by jonasrosland — COMMENTED** [review]
Just one small question, otherwise this looks great, thank you both @stevesloka and @danehans !

File: site/_posts/2020-12-18-contour_v1110.md (hunk 2 above)
Comment (stevesloka) [reply site/_posts/2020-12-18-contour_v1110.md:26]: Nope I think that's right, it shows the plural which is `contours` (yup sounds werid).

File: site/_posts/2020-12-18-contour_v1110.md (hunk 1 above)
Comment (stevesloka) [reply site/_posts/2020-12-18-contour_v1110.md:71]: added a note about that.

File: site/_posts/2020-12-18-contour_v1110.md (hunk 2 above)
Comment (jonasrosland) [reply site/_posts/2020-12-18-contour_v1110.md:26]: Ok great, thanks :)

**Review by jonasrosland — APPROVED** [review]

**Pull request merged**